### ISSUE TITLE
Where are the R communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/r#96]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is R supported by one or more large organizations?
* Does R have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about R? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for R? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in R? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in R?
* Are there popular newsletters for R?
* Is R taught at programming bootcamps? (If so, what are some examples?)
* Is R taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about R?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by zacchaeusluke** [comment]
R is definitely taught at universities. I study Maths at Uni and we did a semester on general R programming in first year. It has come into a lot of other modules as well, predominately statistical ones, and I have a whole module next year on R programming. The maths and statistics communities tend to be enthusiastic about R.

**Comment by jonmcalder** [comment]
- Is R supported by one or more large organizations?

https://www.r-consortium.org/

- Does R have an official community manager?

Not sure about an "official" R community manager, but https://twitter.com/dataandme is a community manager for RStudio and a big influencer in the online community.

- Do you know of specific communities (online or offline) that are enthusiastic about R? (Chat communities, forums, meetups, student clubs, etc)

Many users follow the #rstats hashtag on Twitter
RStudio Community: https://community.rstudio.com/
ROpenSci Community: https://ropensci.org/community/ or https://discuss.ropensci.org/
R User Groups around the world: https://jumpingrivers.github.io/meetingsR/r-user-groups.html

- Are there popular conferences for R? (If so, what are some examples?)

The biggest regular conferences are probably [useR](https://www.r-project.org/conferences.html), [rstudio::conf](https://www.rstudio.com/conference/) and [EARL](https://earlconf.com/), but there are loads more throughout the year - [see here for a list](https://jumpingrivers.github.io/meetingsR/events.html)

- Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in R? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

https://rladies.org/

- Are there specific groups or programs dedicated to mentoring people in R?

http://swirlstats.com/
https://www.jessemaegan.com/post/r4ds-the-next-iteration/

- Are there popular newsletters for R?

https://rweekly.org/
https://www.r-bloggers.com/

- Is R taught at programming bootcamps? (If so, what are some examples?)

https://www.datacamp.com/
https://nycdatascience.com/data-science-bootcamp/
https://datasciencedojo.com/bootcamp/

- Is R taught at universities? (If so, what are some examples?)

[John Hopkins](https://www.jhu.edu/) (also behind a very popular [Coursera course](https://www.coursera.org/learn/r-programming))
[Auckland University](https://www.stat.auckland.ac.nz/en/about/our-research/software-development/sw-the-r-project.html)
[Stanford](http://library.stanford.edu/projects/r)
[Harvard](https://online-learning.harvard.edu/course/data-analysis-life-sciences-1-statistics-and-r-0)

**Comment by kytrinyx** [comment]
This is fantastic, @jonmcalder, thank you!

**Comment by katrinleinweber** [comment]
There are also the Carpentries ([Software](https://github.com/swcarpentry?q=R) & [Data](https://github.com/datacarpentry?q=R)) who introduce mostly master & PhD students to R for scientific applications. They emphasise inclusion and I [started a discussion](https://github.com/swcarpentry/lesson-example/issues/163) to refer their learners to Exercism tracks.

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼

**Comment by katrinleinweber** [comment]
You're welcome :-)
